dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -225,4 +225,56 @@ public void testWorkflowDAGWDLMissingTask() throws IOException, TimeoutException
         //JSON will have no content at all
         Assert.assertEquals("JSON should be blank", strings.size(),0);
     }
+
+    @Test
+    public void testDAGImportSyntax() throws IOException, TimeoutException, ApiException {
+        //Test 'Dockstore.cwl'
+        //will return DAG with two nodes and an edge connecting it, despite there is "import" syntax in "run" field
+
+        final ApiClient webClient = WorkflowET.getWebClient();
```

Comment:
Oh, could you actually re-factor this a bit, putting re-used code into a method?
I just want to make it a bit more clear what is different between the various test cases and what is the same when we next work on this. Thanks!